libfreerdp/codec/dsp.c

```diff
@@ -1254,6 +1270,7 @@ BOOL freerdp_dsp_supports_format(const AUDIO_FORMAT* format, BOOL encode)
 #if defined(WITH_DSP_EXPERIMENTAL)
 
 		case WAVE_FORMAT_ADPCM:
+			return FALSE;
 		case WAVE_FORMAT_DVI_ADPCM:
```

Comment:
Is this still experimental after this fixes?

Reply:
For me it works, but...
Do you mean that "experimental" means "does not work"? :-)

Anyway I let you judge. I understood that part is marked experimental because not well tested (as the most tested part is the FFMPEG part). I just make it work for my usage, I don't guarantee that it works for all use cases compatible with DVI_ADPCM.

About the diff above, I just wanted to flag that the WAVE_FORMAT_ADPCM was not working, and so I wanted to prevent its usage, even in WITH_DSP_EXPERIMENTAL. (The issue with that codec is probably the same as the one I fixed about DVI_ADPCM, but I just needed to make DVI_ADPCM work, so the ADPCM was out of my scope)

Reply:
ok, got it. well, then lets keep it this way and change once we have some field data from test installations :)